### PR TITLE
Enthalpy correction fix

### PR DESCRIPTION
**Pull request by nagelt**
This PR moves a hard coded magic number to input file control. It also fixes some T2HC benchmarks.

### CODE REVIEW

File: FEM/fem_ele_std_tes.cpp [hunk 1]
```diff
@@ -689,6 +689,8 @@ double CFiniteElementStd::CalCoef_RHS_TES(const int dof_index)
 				// AB + \Delta H <--> A + B
 				H_vap = - SolidProp->reaction_enthalpy;
 				//enthalpy correction
+				if (SolidProp->Capacity_mode == 4 || SolidProp->Capacity_mode == 5)
+				{
 				const double rhoSR = gp_ele->rho_s_curr[gp];
```
Comment (norihiro-w) [inline FEM/fem_ele_std_tes.cpp]: please fix indentation from this line

File: FEM/fem_ele_std_tneq.cpp [hunk 2]
```diff
@@ -743,7 +743,10 @@ double CFiniteElementStd::CalCoef_RHS_TNEQ(const int dof_index)
 			Xw = ipol(X0, X1, theta, this);
 			// end of adding eos_arg-----------------
 			double H_vap = - SolidProp->reaction_enthalpy; //sign convention: defined negative for exothermic composition reaction but equ. written as: AB + \Delta H <--> A + B
+
 			//Correction for temperature-dependent enthalpy in case of variable cpS
+			if (SolidProp->Capacity_mode == 4 || SolidProp->Capacity_mode == 5)
+			{
 			const double rhoSR = gp_ele->rho_s_curr[gp];
```
Comment (norihiro-w) [inline FEM/fem_ele_std_tneq.cpp]: same here

Comment (nagelt) [inline FEM/fem_ele_std_tneq.cpp]: clang-format-diff didn't seem to catch this ...

**Comment by norihiro-w** [comment]
@nagelt thanks! this PR fixed all T2HC and TES benchmarks

**Pull request merged**